app/src/main/assets/script.en.md

```diff
@@ -223,6 +258,16 @@ Don't group at all and don't show any headers (regardless of sort order)
         return ""
     end
 
+A callback to modify the display of a task:
+
+    function onDisplay(t,f,e)
+       if f.due~=nil and os.time() > f.due then
+         --- Display overdue tasks in uppercase. (Prefixing with '=' replaces entire task.)
```

Comment:
The intent was to:

Allow the user to _only_ modify the "title" of the task, without having to handle anything else. Tags, recurrences, etc. would all continue to be displayed normally (and colored, where appropriate). This would allow the user to (e.g.) hide all tags (to reduce the amount of text on the screen), but append a '+' to all tasks that have any tags associated with it.

or:

Allow the user (almost) complete control over how the task is displayed. For example, currently, tags are shown in the position that they are entered on the line. As a result, a tag might be shown in the middle of a line or at the end. If the user wanted the tags to _always_ be shown last, the '=' could be used to do so. As another example, due dates are shown as `due:2017-03-20`. By using '=', the user could shorten it to `d:03-20`.

One drawback to using '=' is that it does not allow coloring of the text.

Another drawback is that it might be over-complicating this app with feature-creep.

Reply:
Not voting one way or another, but lua scripts are the one place that I'm most comfortable with feature creep. I think we should not expect anyone except power users to use them -- ie, the app should be fully functional without. 

I also think they're an excellent way to "beta test" features that might eventually make their way into the ui. For example, two stage filtering right now.

Reply:
I prefer giving the user  all building blocks to do whatever he needs without any magic prefix handling.

Reply:
Avoiding a magic prefix sounds sensible. How do you want `onDisplay()` to behave? Should it be limited to modifying only the "title" of the text? Or should it be responsible for handling all the task details? Or should there be some user-settable option to control the display of the task? Does the implementation/functionality need to be re-thought? etc.

Reply:
After the merge I have updated the code how I want it to work. The callback parameters now contain all info  (including the parsed tokens) to write any display Lua function.

Reply:
Again thank you for this pull request. Adding this was on my list for quite some time